etc/kayobe/kolla.yml

```diff
@@ -176,9 +168,9 @@ stackhpc_rocky_9_additional_repos:
   - url: "{{ stackhpc_repo_centos_stream_9_nfv_openvswitch_url }}"
     file: "CentOS-NFV-OpenvSwitch.repo"
     tag: "centos-nfv-openvswitch"
-  - url: "{{ stackhpc_repo_centos_stream_9_openstack_antelope_url }}"
-    file: "CentOS-OpenStack-antelope.repo"
-    tag: "centos-openstack-antelope"
+  - url: "{{ stackhpc_repo_centos_stream_9_openstack_caracal_url }}"
+    file: "CentOS-OpenStack-caracal.repo"
+    tag: "centos-openstack-caracal"
   - url: "{{ stackhpc_repo_centos_stream_9_opstools_url }}"
     file: "CentOS-OpsTools.repo"
     tag: "centos-opstools"
```

Comment:
`stackhpc_repo_rhel_9_treasuredata_4_url` -> `stackhpc_repo_rhel_9_treasuredata_5_url`

and

`stackhpc_repo_rhel_9_mariadb_10_6_url` -> `stackhpc_repo_rhel_9_mariadb_10_11_url`

Reply:
Done